ansible/ansible.cfg

```diff
@@ -8,3 +8,4 @@ forks = 20
 
 [ssh_connection]
 ssh_args = -o StrictHostKeyChecking=no -C -o ControlMaster=auto -o ControlPersist=60s -o ServerAliveInterval=30 -o ServerAliveCountMax=10
+retries=5
```

Comment:
Was gonna mention adding spaces around the `=`, then I noticed the file mixes these a bunch, so meh (we do need to get better at keeping coherent styles though).

Reply:
Why not, lets unify the formatting.